pkg/source/psql/backend_config_based.go

```diff
@@ -0,0 +1,58 @@
+package psql
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+	"os"
+
+	"github.com/mittwald/brudi/pkg/cli"
+)
+
+type ConfigBasedBackend struct {
+	cfg *Config
+}
+
+func NewConfigBasedBackend() (*ConfigBasedBackend, error) {
+	config := &Config{
+		&Options{
+			Flags:          &Flags{},
+			AdditionalArgs: []string{},
+			SourceFile:     "",
+		},
+	}
+
+	err := config.InitFromViper()
+	if err != nil {
+		return nil, err
+	}
+
+	return &ConfigBasedBackend{cfg: config}, nil
+}
+
+func (b *ConfigBasedBackend) RestoreBackup(ctx context.Context) error {
+	fileName, err := cli.CheckAndGunzipFile(b.cfg.Options.SourceFile)
+	args := append(cli.StructToCLI(b.cfg.Options.Flags), b.cfg.Options.AdditionalArgs...)
+	args = append(args, []string{fmt.Sprintf("--command=\\i %s", fileName)}...)
```

Comment:
Why are you not using the `Command` property of the `Flags` struct for appending `--command`?   
Doing it via property you will receive a perfect build command when calling `cli.StructToCLI`.